packages/node/src/types.ts

```diff
@@ -2,6 +2,8 @@ import { ClientOptions, Options, TracePropagationTargets } from '@sentry/types';
 
 import { NodeTransportOptions } from './transports';
 
+type SentryNodeInstrumenter = 'sentry' | 'otel';
```

Comment:
I guess we'll need to move this to `@sentry/types` eventually, as the otel package will also need this. I'll do this in the next step, and prob. expose this as `Instrumenter`?